test/core/store/test_typeid.py

```diff
@@ -0,0 +1,102 @@
+import geopandas as gpd
+import unittest
+import xarray as xr
+
+from xcube.core.new import new_cube
+from xcube.core.store.typeid import TYPE_ID_CUBE
+from xcube.core.store.typeid import TYPE_ID_DATASET
+from xcube.core.store.typeid import TYPE_ID_GEO_DATA_FRAME
+from xcube.core.store.typeid import TYPE_ID_MULTI_LEVEL_DATASET
+from xcube.core.store.typeid import get_type_id
+from xcube.core.store.typeid import TypeId
+
+from xcube.core.mldataset import BaseMultiLevelDataset
+
+
+class TypeIdTest(unittest.TestCase):
```

Comment:
In `TypeIdTest` test methods, please use names and flags that apply to our context, so we (and others) can better grasp the intention of `TypeId`.